src/components/menus/ActivityStatusMenu.svelte

```diff
@@ -0,0 +1,93 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import WaterfallIcon from '@nasa-jpl/stellar/icons/waterfall.svg?component';
+  import { createEventDispatcher } from 'svelte';
+  import type { ActivityDirectiveValidationStatus } from '../../types/activity';
+  import type { ActivityErrorCounts } from '../../types/errors';
+  import { Status } from '../../utilities/status';
+  import PlanNavButton from '../plan/PlanNavButton.svelte';
+  import ActivityErrorsRollup from '../ui/ActivityErrorsRollup.svelte';
+
+  export let activityErrorCounts: ActivityErrorCounts;
+  export let activityDirectiveValidationStatuses: ActivityDirectiveValidationStatus[] = [];
+  export let compactNavMode: boolean = false;
+
+  const dispatch = createEventDispatcher();
+
+  let invalidActivityCount: number = 0;
+  let totalActivitiesCheckedCount: number = 0;
+  let totalActivityCount: number = 0;
+
+  $: {
+    ({ invalidActivityCount, totalActivitiesCheckedCount, totalActivityCount } =
+      activityDirectiveValidationStatuses.reduce(
+        (prevCounts, validationStatus) => {
+          return {
+            invalidActivityCount:
+              prevCounts.invalidActivityCount + (validationStatus.validations.success === false ? 1 : 0),
+            totalActivitiesCheckedCount:
+              prevCounts.totalActivitiesCheckedCount + (validationStatus.status === 'complete' ? 1 : 0),
+            totalActivityCount: prevCounts.totalActivityCount + 1,
+          };
+        },
+        {
+          invalidActivityCount: 0,
+          totalActivitiesCheckedCount: 0,
+          totalActivityCount: 0,
+        },
+      ));
+  }
+
+  function onClickViewConsole() {
+    dispatch('viewActivityValidations');
+  }
+</script>
+
+<PlanNavButton
+  title={!compactNavMode ? 'Activities' : ''}
+  menuTitle="Activity Status"
+  showStatusInMenu={false}
+  statusBadgeText={`${invalidActivityCount}`}
+  status={activityErrorCounts.all && activityErrorCounts.all > 0 ? Status.Failed : Status.Complete}
+>
+  <WaterfallIcon />
+  <svelte:fragment slot="metadata">
+    <div class="activity-status-nav-container">
+      <div class="total-count">
+        {totalActivitiesCheckedCount}/{totalActivityCount} activit{totalActivityCount !== 1 ? 'ies' : 'y'} checked
+      </div>
+      <div class="invalid-count">
+        {invalidActivityCount} activit{totalActivityCount !== 1 ? 'ies' : 'y'} have problems
```

Comment:
Could we display a different styling/message if 0 activities have problems?
<img width="253" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/d2b0f822-9326-4ff3-bb51-ba3b3ad14a15">

Like "No problems detected" or something?

Reply:
Thanks @AaronPlave ! I made it "No problems detected" and colored green.